Probesys/bileto
# dev: Refactor the ticket forms

## Related issue(s)



https://github.com/Probesys/bileto/issues/551

## How to test manually



1. Use the ticket forms
2. Verify that everything work

## Checklist
